main/chains/rpchProvider/index.ts

```diff
@@ -0,0 +1,222 @@
+import RPChSDK, { type Ops } from '@rpch/sdk'
+import { isError } from '@rpch/sdk/build/jrpc'
+import EventEmitter from 'events'
+import { Connection } from 'ethereum-provider/dist/types'
+import { v4 as uuid } from 'uuid'
+import provider from 'eth-provider'
+import store from '../../store'
+import log from 'electron-log'
+
+// @ts-ignore
+import ProviderCreator from 'eth-provider/provider'
+import { DISCOVERY_PLATFORM_API_ENDPOINT, RPCH_SECRET_TOKEN } from '../../config'
+
+const dev = process.env.NODE_ENV === 'development'
+
+class RPChSDKSingleton {
+  static sdk: RPChSDK | undefined
+
+  static options: Ops = {
+    discoveryPlatformEndpoint: DISCOVERY_PLATFORM_API_ENDPOINT || undefined,
+    forceZeroHop: true
+
+    // TODO: Remove after confirmation and testing
+    // debugScope: 'rpch:*'
+  }
+
+  static send(...args: Parameters<RPChSDK['send']>): ReturnType<RPChSDK['send']> {
+    if (!this.sdk) {
```

Comment:
in a static class accessing the static properties should always be done with the class name to indicate static access, e.g.: `if (!this.sdk) {` -> `if (!RPChSDKSingleton.sdk) {`

Reply:
Fixed.